cmake/stm32/f3.cmake

```diff
@@ -4,19 +4,19 @@ set(STM32_F3_TYPES
     F378xx F398xx 
 )
 set(STM32_F3_TYPE_MATCH 
-    "301.[68]" "302.[68]" "302.[BC]" "302.[ED]" "303.[68]" "303.[BC]" 
-    "303.[ED]" "318.." "328.." "334.[468]" "358.." "373.[8BC]"
+    "301.[68]" "302.[68]" "302.[BC]" "302.[DE]" "303.[68]" "303.[BC]" 
+    "303.[DE]" "318.." "328.." "334.[468]" "358.." "373.[8BC]"
     "378.." "398.."    
 )
 set(STM32_F3_RAM_SIZES 
-     16K  16K   0K  64K  16K   0K  
-     80K  16K  16K  16K  48K   0K
-     32K  80K
+     16K  16K   0K  64K  12K   0K  
+     64K  16K  12K  12K  40K   0K
```

Comment:
The first one is 80Ko
![image](https://user-images.githubusercontent.com/29129731/102714777-39305900-42d1-11eb-9e93-c828fed2f801.png)
4th
![image](https://user-images.githubusercontent.com/29129731/102714974-895beb00-42d2-11eb-9559-d82976fe6d61.png)
The last one :
![image](https://user-images.githubusercontent.com/29129731/102714910-3aae5100-42d2-11eb-9190-4670f4bb84bc.png)

Reply:
From my understanding:
- F303.[DE] 80 = 64+16 non aliased 
- F318.. 16 this seems OK with you capture !?!?
- F318.. 16 = 12 + 4 non aliased
- F358.. 48 = 40+8 non aliased
- F373.. it's mentioned "up to" so it depends on last letter. You should look at bottom of the file for sizes